projects/cashmere/src/lib/app-switcher/app-switcher.component.html

```diff
@@ -1,26 +1,26 @@
-<div #brand
-     [popperTarget]="brand"
-     [hcPopover]="appSwitcherContent"
-     popperPlacement="bottom-start"
-     [className]="brandBg"
-     (popperOnShown)="buttonOn()"
-     (popperOnHidden)="buttonOff()">
-     <img src="{{brandIcon}}">
-</div>
-
-<hc-popover-content #appSwitcherContent>
-    <div class="popover-content">
-        <h4>Applications</h4>
-        <pre *ngIf="!applications">loading applications...</pre>
-        <div class="apps">
-            <div class="app" *ngFor="let app of applications">
-                <a class="thumbnail" title="{{app.Description}}" href="{{app.ServiceUrl}}" target="_blank">
-                    <img class="popover-icon img-responsive" src={{app.Icon}}>
-                    <div class="title">{{app?.FriendlyName | ellipsis:18}}</div>
-                </a>
-            </div>
-        </div>
-        <a class="all-apps-link" href={{appSwitcherService.allApplicationsUri}} target="_blank">View all my
-            applications</a>
-    </div>
-</hc-popover-content>
+<div #brand
+     [popperTarget]="brand"
+     [hcPopover]="appSwitcherContent"
+     popperPlacement="bottom-start"
+     [className]="brandBg"
+     (popperOnShown)="buttonOn()"
+     (popperOnHidden)="buttonOff()">
+     <img src="{{brandIcon}}">
+</div>
+
+<hc-popover-content #appSwitcherContent>
+    <div class="popover-content">
+        <h4>Applications</h4>
+        <pre *ngIf="!applications">loading applications...</pre>
+        <div class="apps">
+            <div class="app" *ngFor="let app of applications">
+                <a class="thumbnail" title="{{app.Description}}" href="{{app.ServiceUrl}}" target="_blank">
+                    <div class="thumbnail-img" style="background-image: url('{{app.Icon}}')"></div>
```

Comment:
replace this line with the following:
```suggestion
                    <div class="thumbnail-img" [style.background-image]="'url(' + app.icon + ')'"></div>
```

Reply:
Thanks @joeskeen !